packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

```diff
@@ -145,52 +150,56 @@ You need to add a DNS TXT record to your domain to verify ownership of your doma
 
 3. Add a TXT record and paste the value that you copied from the **Host** column into the appropriate field, for example, the **Name** or **Host** field.
```

Comment:
You also need to add a CNAME record for BYOC, not just TXT.  This is not a separated section in the wizard anymore in the new UI.

Reply:
I did move the CNAME content up to immediately following the DNS TXT section. You're right, though, I need to clarify that both are done at the same point of the process :)